pkg/apis/servicecatalog/validation/binding.go

```diff
@@ -52,6 +52,20 @@ func validateBindingSpec(spec *sc.BindingSpec, fldPath *field.Path, create bool)
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("secretName"), spec.SecretName, msg))
 	}
 
+	if spec.AlphaPodPresetName != nil {
```

Comment:
Unfortunately we do not export validations at all in client-go.  I opened https://github.com/kubernetes/kubernetes/issues/44995 to try to figure something out about this.